vitalii-vov/Maui.PDFView
# Sam mapssiv 1316 prototype support in standalone

![image](https://github.com/vitalii-vov/Maui.PDFView/assets/40006493/5c5ddc68-950e-4bea-ad67-6dc33f46d976)
![image](https://github.com/vitalii-vov/Maui.PDFView/assets/40006493/0e004f42-a8f2-4ed7-b829-31508fdadcc3)

Path to Navy PDFs: 
https://github.com/MApSS-DEV/mynavy_uniforms-content/tree/main/PDFs

To test PDFs, 

- rename the PDFs you are testing to pdf1 or pdf2 (See Example/Resources/PDF)
- verify the Example.proj has the following listings

- -	<ItemGroup>
		<None Remove="Resources\PDF\pdf1.pdf" />
		<None Remove="Resources\PDF\pdf2.pdf" />
	</ItemGroup>

	<ItemGroup>
		<EmbeddedResource Include="Resources\PDF\pdf1.pdf" />
		<EmbeddedResource Include="Resources\PDF\pdf2.pdf" />
	</ItemGroup>

PDF __pdf2.pdf will crash the system someplace in the XAML. No error message provided
